content/docs/command-reference/exp/run.md

```diff
@@ -73,16 +75,22 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
 
 - `-S [<filename>:]<override_pattern>`,
   `--set-param [<filename>:]<override_pattern>` - set the value of existing
-  `dvc params` for this experiment. `filename` can be any valid params file
-  (`params.yaml` by default). This will override the param file before running
-  the experiment.
+  `dvc params` for this experiment.
 
-  For example, to override the value an existing param, you can use
-  `--set-param <param_name>=<param_value>`.
+  This will update the param file **before** running the experiment.
 
-  Valid `<override_pattern>` values are defined in the
-  [Hydra Override Grammar](https://hydra.cc/docs/advanced/override_grammar/basic/#modifying-the-config-object).
-  Patterns modifying Hydra's defaults list are not supported.
+  For example, to modify the value of an existing param, you can use
+  `-S <name>=<value>`.
```

Comment:
Maybe better to have an actual basic example in the Examples section to shorten this a bit.

Reply:
I am ok with shorting. Do you think the basic example is necessary? It's already present in the Get Started and User guide

Reply:
IMO I think it's useful to emphasize the basic syntax somewhere in the command ref since it's not obvious otherwise, but it's not a blocker, nor do I have that strong an opinion on where it goes.